server/gokbg3/grails-app/services/org/gokb/UserProfileService.groovy

```diff
@@ -58,8 +60,111 @@ class UserProfileService {
       user_to_delete.delete(flush: true, failOnError: true)
 
       log.debug("Done")
+      result.result = "OK"
     } else {
       log.error("Could not find either the user object for deletion (${params.id}) or the placeholder user")
+      result.result = "ERROR"
+      result.errors = user_to_delete.errors
```

Comment:
This will throw an exception if `user_to_delete == null`

Reply:
done